content/lectures/final_project.md

```diff
@@ -0,0 +1,39 @@
+Title: Final Project Instructions
+Summary:  Final Project
+Date: 10/31/2017
+Tags: final, project
+Category: lectures
+Author: Kerry Key and Ryan Abernathey
+
+
+
+** Final Project **
+
+
+The final project is intended to be an independent project that uses the skills you have learned in this class, as well as things you have learned on your own, to create a complicated and interesting data analysis program that accomplishes one or more research-relevant tasks. Unlike the homework assignments where we supplied the data files and told you what to do, now it is your turn to identify a  data set relevant to your own research interests and to create a code that will analyze that data and present  results in graphical form.
+
+***Grading***
+
+The final project will be worth 40% of your class grade so you should expect to put a good amount of time and effort into it.  
+
+ In order to get the maximum credit for the final project you will need to:
+
+1. Turn in a project proposal by **Thursday, November 9**. The proposal description only needs to be a paragraph long. It should describe in words what your final project will do and what programming languages you intend to use. We will review your proposal ASAP and let you know if it is acceptable.  Place your proposal description in a file call `proposal.txt` and turn it via github in a folder call `FinalProject`.
```

Comment:
We should allow them to write markdown if they prefer.

Reply:
sounds good